addons/website_slides/views/slide_channel_views.xml

```diff
@@ -194,6 +192,7 @@
                 <search string="Courses">
                     <field name="name" string="Course"/>
                     <filter string="Archived" name="inactive" domain="[('active','=',False)]"/>
+                    <filter string="Published" name="published" domain="[('is_published', '=', True)]"/>
```

Comment:
Name should be ``filter_published`` . I know the one just above is badly named but hey ...